build-logic/convention/src/main/kotlin/AndroidApplicationFirebaseConventionPlugin.kt

```diff
@@ -35,7 +36,10 @@ class AndroidApplicationFirebaseConventionPlugin : Plugin<Project> {
                 val bom = libs.findLibrary("firebase-bom").get()
                 add("implementation", platform(bom))
                 "implementation"(libs.findLibrary("firebase.analytics").get())
-                "implementation"(libs.findLibrary("firebase.performance").get())
+                "implementation"(libs.findLibrary("firebase.performance").get()) {
+                    exclude(group = "com.google.protobuf", module = "protobuf-javalite")
```

Comment:
Please add a comment as to why these exclusions are necessary (are they definitely necessary with all the version changes, maybe this isn't currently a problem?)